website/docs/guides/intro.md

```diff
@@ -15,38 +15,41 @@ the [cookbooks section](../cookbooks/intro.md).
 
 These few guides are a good starting point when learning how to use Scala CLI.
 
-- [Configuration](./configuration.md) - learn how to configure various options, and what configuration styles are best
+- [Configuration](introduction/configuration.md) - learn how to configure various options, and what configuration styles are best
```

Comment:
It's hard to figure out now, what is the difference between a guide and cook book. Isn't it basically the same?

Reply:
Good point, the difference is quite slim, maybe we should consider merging them?
Although some of them won't really fit the guides, those could be left in a separate `Examples` section.
WDYT?
Of course it's for a potential separate ticket.

Reply:
alternatively, we could indeed merge them and only have guides... dunno, might be worth discussing it